.travis.yml

```diff
@@ -4,7 +4,8 @@ os:
 
 language: node_js
 node_js:
-  - "6"
```

Comment:
Why did we drop 6?

Reply:
Node 6 is not dropped, it's still below in the test matrix settings. Only the ``testBuildIntegrity`` test is run with Node 8 and 9, all the other tests are run against Node 6. So all the test from before are still running with the same environment, with ``testBuildIntegrity`` (just running a single state test) we are able to do additional checks without using too much of the limited run time for tests in travis.

Reply:
Cool, travis config can be misleading in cases, especially when the section shown by github is small :)

Reply:
So the goal of `testBuildIntegrity` is to have a quick check with a single test?

Reply:
Yes, to catch stuff like "is Node 10" generally working, "is the dist folder existing", stuff like that.